boards/arm/nrf9160dk_nrf52840/doc/index.rst

```diff
@@ -123,37 +123,116 @@ Board controller firmware
+| P0.26, P0.27, P0.28, and P0.29 | VCOM0                            |
++--------------------------------+----------------------------------+
+| P0.01, P0.00, P0.15, and P0.14 | VCOM2                            |
++--------------------------------+----------------------------------+
+| P0.02                          | LED1                             |
++--------------------------------+----------------------------------+
+| P0.03                          | LED2                             |
++--------------------------------+----------------------------------+
+| P0.04                          | LED3                             |
++--------------------------------+----------------------------------+
+| P0.05                          | LED4                             |
++--------------------------------+----------------------------------+
+| P0.08                          | Switch 1                         |
++--------------------------------+----------------------------------+
+| P0.09                          | Switch 2                         |
++--------------------------------+----------------------------------+
+| P0.06                          | Button 1                         |
++--------------------------------+----------------------------------+
+| P0.07                          | Button 2                         |
++--------------------------------+----------------------------------+
+| P0.17, P0.18, and P0.19        | Arduino pin headers              |
++--------------------------------+----------------------------------+
+| P0.21, P0.22, and P0.23        | Trace interface                  |
++--------------------------------+----------------------------------+
+| COEX0, COEX1, and COEX2        | COEX interface                   |
++--------------------------------+----------------------------------+
+
+For a complete list of all the routing options available,
+see the `nRF9160 DK board control section in the nRF9160 DK User Guide`_.
+
+If you want to route some of the above pins differently or enable any of the
+other available routing options, enable or disable the devicetree node that
+represents the analog switch that provides the giving routing.
+For example, if you want to enable the optional routing for the nRF9160 pins
+P0.17, P0.18, and P0.19 so that they are routed to nRF52840 pins P0.17, P0.20,
+and P0.15, respectively, add the following in the devicetree overlay in your
+application:
+
+.. code-block:: none
+
+   &nrf_interface_pins_0_2_routing {
+           status = "okay";
+   };
+
+And if you want to, for example, disable routing for the VCOM2 pins, add the
+following:
+
+.. code-block:: none
+
+   &vcom2_pins_routing {
+           status = "disabled";
+   };
+
+
+The following devicetree nodes are defined for the analog switches present
+on the nRF9160 DK:
+
++------------------------------------+------------------------------+
+| Devicetree node alias              | Analog switch name           |
```

Comment:
Please use "node label" instead of "node alias", since these are not in `/aliases`.

Reply:
Sure, I meant the node label but I used a wrong term.